.vscode/settings.json

```diff
@@ -1,7 +1,8 @@
 {
-  "git.ignoreLimitWarning": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnPaste": false,
-  "editor.formatOnSave": true,
-  "editor.wordWrap": "on"
+	"git.ignoreLimitWarning": true,
+	"editor.defaultFormatter": "esbenp.prettier-vscode",
+	"editor.formatOnPaste": false,
+	"editor.formatOnSave": true,
+	"editor.wordWrap": "on",
```

Comment:
This setting should **not** be included in the project settings. This is a user preference and no point in enforcing it for the whole team. Also one of my absolute least favourite things about VSCode is that you literally cannot override project settings with your user config, even for cosmetic things like this. If we remove this from here this will just mean that everyone can configure it how they like.

```suggestion
```